tests/api2/test_260_iscsi.py

```diff
@@ -546,6 +549,7 @@ def test_54_redisconnect_iscsi_zvol_target(request):
     cmd = f'iscsictl -R -t {basename}:{zvol_name}'
     results = SSH_TEST(cmd, BSD_USERNAME, BSD_PASSWORD, BSD_HOST)
     assert results['result'], f"{results['output']}, {results['stderr']}"
+    sleep(2)
```

Comment:
Are the sleeps being added to help with stability?

Reply:
Yes, sometimes we do issues with the next test because the code runs faster than the system can handle.